nextflow_schema.json

```
@@ -48,24 +48,34 @@
                 },
                 "min_counts": {
                     "type": "integer",
-                    "description": "Minimum number of variant observations required.",
+                    "description": "minimum counts for variant to be recognized. All variants below min_counts will be set to 0",
                     "minimum": 1,
                     "default": 3
                 },
+		"sliding_window_size": {
+                    "type": "integer",
+                    "description": "To flatten graphs in plots (e.g. GLOBAL_POS_BIASES_COUNTS function)",
+                    "default": 10
+                },
+		"aimed_cov": {
+                    "type": "integer",
+                    "description": "aimed coverage (assuming equal spread) to visualize threshold in plots",
+                    "default": 100
+                },
                 "mutagenesis_type": {
                     "type": "string",
-                    "description": "Type of mutagenic primers. Choose from nnk, nns, max_diff_to_wt, custom. When using 'custom', also provide the parameter 'custom_codon_library'",
+                    "description": "Type of mutagenic primers. Choose from nnk, nns, max_diff_to_wt, custom. When using 'custom', also provide the parameter 'custom_codon_library'. Default library is set to nnk",
```

```suggestion
                    "description": "Type of mutagenic primers. Choose from nnk, nns, max_diff_to_wt, custom. When using 'custom', also provide the parameter 'custom_codon_library'.",
```
we usually show the default value in all the relevant places already.